modules/ztp-configuring-ipsec-using-ztp-and-siteconfig-for-mno.adoc

```diff
@@ -0,0 +1,107 @@
+
+* You have created a Git repository where you manage your custom site configuration data.
+The repository must be accessible from the hub cluster and be defined as a source repository for the Argo CD application.
+
+* You have installed the `butane` utility, version 0.20.0 or higher.
+
+* You have a PKCS#12 certificate for the IPsec endpoint and a CA cert in PEM format.
+
+* You have installed the NMState Operator
+
+.Procedure
+
+. Run the `import-certs.sh` script that creates a `MachineConfig` CR to import the external certs:
++
+If the PKCS#12 certificate is protected with a password, set the `-W` argument.
++
+.Example output
+[source,terminal]
+----
+TBD
+----
+
+. Include `99-ipsec-master-import-certs.yaml` and `99-ipsec-worker-import-certs.yaml` files in one of the additional install-time manifests directories.
+
+. Configure the `ipsec-config-policy.yaml` with the required values to create an IPsec configuration with the NMState Operator's node network configuration policy:
++
+[source,yaml]
+----
+apiVersion: policy.open-cluster-management.io/v1
+kind: ConfigurationPolicy
+metadata:
+  name: policy-config
+spec:
+  namespaceSelector:
+    include: ["default"]
+    exclude: []
+    matchExpressions: []
+    matchLabels: {}
+  remediationAction: inform
+  severity: low
+  evaluationInterval:
+    compliant:
+    noncompliant:
+  object-templates-raw: |
+    {{- range (lookup "v1" "Node" "" "").items }}
+    - complianceType: musthave
+      objectDefinition:
+        kind: NodeNetworkConfigurationPolicy
+        apiVersion: nmstate.io/v1
+        metadata:
+          name: {{ .metadata.name }}-ipsec-policy
+        spec:
+          nodeSelector:
+            kubernetes.io/hostname: {{ .metadata.name }}
+          desiredState:
+            interfaces:
+            - name: hosta_conn
+              type: ipsec
+              libreswan:
+                left: <cluster_node> <1>
+                leftid: '%fromcert'
+                leftmodecfgclient: false
+                leftcert: <left_cert> <2>
+                leftrsasigkey: '%cert'
+                right: <external_host> <3>
+                rightid: '%fromcert'
+                rightrsasigkey: '%cert'
+                rightsubnet: <external_address> <4>
+                ikev2: insist <5>
+                type: tunnel
+----
+<1> Replace `<cluster_node>` with the IP address or DNS hostname of the cluster node for the cluster-side IPsec tunnel.
+<2> Replace `<left_cert>` with the IPsec certificate nickname.
+<3> Replace `<external_host>` with the external host IP address or DNS hostname.
+<4> Replace `<external_address>` with the IP address or subnet of the external host on the other side of the IPsec tunnel.
+<5> Use the IKEv2 VPN encryption protocol only. Do not use IKEv1, which is deprecated.
+
+. Add your `ca.pem` and `left_server.p12` certificates to the `optional-extra-manifest/ipsec` folder.
```

Comment:
We are pointing here to left_server certificate while in the previous section we suggest replacing it, should we simply point to left_server in the previous section as well? also to follow the official IPsec documentation:  https://docs.openshift.com/container-platform/4.16/networking/network_security/configuring-ipsec-ovn.html#nw-ovn-ipsec-north-south-enable_configuring-ipsec-ovn

Reply:
Updated as per the suggestion.